README.rst

```diff
@@ -1,6 +1,6 @@
 
-.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
```

Comment:
we are using GitHub actions for CI now, so if you can figure out how to get the URL for that badge we would like it here instead of the travis one you deleted.